item-search/README.md

```diff
@@ -105,13 +105,13 @@ See [examples](examples.md) for some example requests.
 
 **limit** The limit parameter follows the same semantics of the OAFeat Item resource limit parameter. The value is
 a suggestion to the server as to the maximum number of Item objects the
-client would prefer in the response. The OpenAPI specification defines the default and maximum values
+client would prefer in the response. The server must return fewer Item objects, but
```

Comment:
It is `<=` and not `<`, right?

Reply:
fixed the wording.  it **may** return fewer, but **must not** return more.

Reply:
Thanks.